mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx

```diff
@@ -0,0 +1,47 @@
+import { ThemedScrollView, ThemedText } from '@components/themed'
+import { AccordionContent, WalletAccordion } from '@components/WalletAccordion'
+import { StackScreenProps } from '@react-navigation/stack'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { LoanParamList } from '../LoansNavigator'
+import { useTnC } from './LoansTnCText'
+
+type Props = StackScreenProps<LoanParamList, 'LoansTnC'>
+
+export function LoansTnC ({ route }: Props): JSX.Element {
+  const faqContent: AccordionContent[] = [
+    {
+      title: translate('components/LoansFaq', 'Loan terms'),
+      content: [{
+        text: translate('components/LoansFaq', useTnC()),
+        type: 'paragraph'
+      }]
+    }
+  ]
+
+  return (
+    <ThemedScrollView
+      contentContainerStyle={tailwind('p-6 pb-8')}
+      testID='loans_faq'
```

Comment:
on `InfoText` there is a `testId`, why is it called here `testID`?

Reply:
This is an identifier for tests (which we currently are not doing and not really planned as really a pain in the a** with merging and our current and future code changes).
I'll try to still keep em in, _just in case_: